src/Makefile.am.inc

```diff
@@ -112,6 +112,7 @@ EXTRA_DIST += \
 
 libexec_PROGRAMS = \
 	xdg-desktop-portal-gtk \
+	pdftoraw \
```

Comment:
Lets prefix this to ensure it never conflicts with anybody else: `xdg-desktop-portal-gtk-pdftoraw`.

Reply:
Or put it in a subdirectory, but `xdg-desktop-portal-gtk` already existed.

Maybe `$libexec/xdg-desktop-portal-gtk-utils/pdftoraw` is nicer.